src/ocean_emulators/eval.py

```diff
@@ -100,6 +100,7 @@ def __init__(self, cfg: EvalConfig) -> None:
         raw = DataSource.from_config(cfg, use_dask=True)
         self.src = validate_data(raw)
         self.data = self.src.data
+        self.static_data = self.src.get_static_data()
```

Comment:
The more pythonic thing to do would be to make `static_data` a property; a getter method is more Java like.  I recommend using a `cached_property`, if possible.

Reply:
Ok, looking at the code below, I think a better fit would be to pass in the `cfg.data.static_data_vars` into the class, similar to `boundary_var_names` and then call: 
```suggestion
        self.static_data = self.src.data[static_data_vars]
```

Reply:
~~I will also let @jder review this line~~